docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+---
+services:
+  docker-linuxserver-ubuntu-fips:
+    image: ${ECR_ACCOUNT_ID:-0123456789012}.dkr.ecr-fips.us-east-1.amazonaws.com/docker-linuxserver-ubuntu-fips:latest
+    pull_policy: build
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: linuxserver-base
+      platforms:
+        - linux/amd64
+      args:
+        - BUILD_DATE=${BUILD_DATE:-2025-07-01T00:00:00Z}
```

Comment:
Should this be updated to today's date? Or just for local dev and doesn't matter too much

Reply:
its just for local dev and doesnt matter at all